app/tests/spec/lib/app-start.js

```diff
@@ -452,11 +487,12 @@ function (chai, sinon, Raven, AppStart, Session, Constants, p, Url, OAuthErrors,
 
       it('migrates data stored in `localStorage.uuid` to `localStorage.uniqueUserId`', function () {
         var storage = Storage.factory('localStorage', windowMock);
-        storage.set('uuid', 'stored in uuid');
+        var uniqueUserId = 'originally stored in uuid';
+        storage.set('uuid', uniqueUserId);
 
         appStart.initializeUniqueUserId();
 
-        assert.equal(storage.get('uniqueUserId'), 'stored in uuid');
+        assert.equal(storage.get('uniqueUserId'), uniqueUserId);
         assert.isUndefined(storage.get('uuid'));
```

Comment:
http://www.w3.org/TR/webstorage/#dom-storage-getitem, the get method should be returning null, does isUndefined include null?